ext/crates/fp/src/prime/mod.rs

```diff
@@ -158,6 +162,20 @@ macro_rules! def_prime_static {
                 $pn::try_from(p).map_err(D::Error::custom)
             }
         }
+
+        #[cfg(feature = "proptest")]
+        impl proptest::arbitrary::Arbitrary for $pn {
+            type Parameters = Option<NonZeroU32>;
+            type Strategy = proptest::strategy::Just<$pn>;
+
+            fn arbitrary_with(_max: Self::Parameters) -> Self::Strategy {
+                // This doesn't honor the max parameter, but that should be fine as long as the
+                // static primes are small enough and/or the max is large enough.
+                proptest::strategy::Just($pn)
```

Comment:
You could presumably return an empty strategy if the max is too small, but it's probably not worth it.

Reply:
If I understand correctly it's actually impossible for a strategy not to give at least one value. Unless it counts everything as a reject, but then that causes local failures